src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -0,0 +1,129 @@
+        }
+
+        const paymentMethod = this._store.getState().paymentMethods.getPaymentMethod(options.methodId);
+
+        if (!paymentMethod) {
+            throw new MissingDataError(MissingDataErrorType.MissingPaymentMethod);
+        }
+
+        return this._stripeScriptLoader.load(paymentMethod.initializationData.stripePublishableKey)
+            .then((stripeJs: StripeV3Js) => {
+                this.stripeJs = stripeJs;
+                const elements = this.stripeJs.elements();
+                this.cardElement = elements.create('card', {
+                    style: DOMElement.elementProps,
+                });
+
+                if (!this.cardElement) {
+                    throw new InvalidArgumentError('Unable to initialize payment because "StripeCardElement" argument is not provided.');
+                }
+
+                this.cardElement.mount('#' + DOMElement.containerId);
+
+                return Promise.resolve(this._store.getState());
+            });
+    }
+
+    execute(payload: OrderRequestBody, options?: PaymentRequestOptions): Promise<InternalCheckoutSelectors> {
+        const { payment, ...order } = payload;
+
+        if (!payment) {
+            throw new PaymentArgumentInvalidError(['payment.paymentData']);
+        }
+
+        return this._store.dispatch(this._paymentMethodActionCreator.loadPaymentMethod(payment.methodId))
+            .then(state => {
+                const paymentMethod = state.paymentMethods.getPaymentMethod(payment.methodId);
+
+                if (!paymentMethod || !paymentMethod.clientToken) {
+                    throw new MissingDataError(MissingDataErrorType.MissingPaymentMethod);
+                }
+
+                return this._getStripeJs().handleCardPayment(
+                    paymentMethod.clientToken, this._getCardElement(), {}
+                ).then((stripeResponse: StripeResponse) => {
+                    if (stripeResponse.error) {
+                        throw new StandardError(stripeResponse.error.message);
+                    } else {
+                        if (!stripeResponse.paymentIntent.id) {
+                            throw new PaymentArgumentInvalidError(['paymentIntent.id']);
+                        }
+
+                        const paymentPayload = {
+                            methodId: payment.methodId,
+                            paymentData: { nonce: stripeResponse.paymentIntent.id },
+                        };
+
+                        return this._store.dispatch(this._orderActionCreator.submitOrder(order, options))
+                            .then(() =>
+                                this._store.dispatch(this._paymentActionCreator.submitPayment(paymentPayload))
+                            );
+                    }
+                });
+            })
+            .catch((error: Error) => { throw new StandardError(error.message); });
```

Comment:
You don't actually need to do this unless you want to throw a specific error type.